litestar/config/app.py

```diff
@@ -61,6 +61,8 @@ class AppConfig:
     The final attribute values are used to instantiate the application object.
     """
 
+    after_shutdown: list[Callable] = field(default_factory=list)
```

Comment:
These wouldnt be `list[LifespanHook]`?

Reply:
I originally had them as that.  It's not really part of the ASGI lifespan and I don't think these can be async functions, but it would be nice to leverage that type if we can.